example/lib/theme/app_decoration.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:equatable/equatable.dart';
+import 'package:flutter/widgets.dart';
+
+class AppDecoration extends Equatable {
```

Comment:
Honestly, don't really like `AppDecoration` as it confuses me a bit - what kind of decorations? Why color / typography are not a part of decoration? And so on.
Maybe rename it to `AnimationDecoration` or something that indicates that those are Animation option?

Reply:
Renamed to `AppAnimation` in https://github.com/getkevin/kevin-flutter/pull/20/commits/205dce361bc77742a6c6fc88391c686ae645f87a